Ref: https://github.com/golang/protobuf/issues/1596#issuecomment-1981208282

php_generic_services field was dropped, so remove it from the testdata

diff --git a/private/bufpkg/bufimage/bufimagefuzz/corpus/alloptions.txtar b/private/bufpkg/bufimage/bufimagefuzz/corpus/alloptions.txtar
index b03fa8a0..9359bf91 100644
--- a/private/bufpkg/bufimage/bufimagefuzz/corpus/alloptions.txtar
+++ b/private/bufpkg/bufimage/bufimagefuzz/corpus/alloptions.txtar
@@ -16,5 +16,4 @@ option cc_generic_services = false;
 option java_generic_services = false;
 option java_multiple_files = false;
 option java_string_check_utf8 = false;
-option php_generic_services = false;
 option py_generic_services = false;
